tensorrt_llm/_torch/modules/fused_moe/moe_prefetch_manager.py

```
@@ -0,0 +1,178 @@
+
+    def get_dst_device_buffers(self, layer_idx: int):
+        assert layer_idx in self.layer_to_buffer, f"Layer {layer_idx} not found in registered prefteched layers"
+        buffer_idx = self.layer_to_buffer[layer_idx]
+        return self.device_weight_buffers[buffer_idx][
+            'w3_w1_weight'], self.device_weight_buffers[buffer_idx]['w2_weight']
+
+    def get_buffer_n_source(self, layer_idx: int):
+        assert layer_idx in self.prefetch_layer_indices, f"Layer {layer_idx} not found in registered prefteched layers"
+        prefetch_layer_idx = self.next_prefetch_layer[layer_idx]
+        if prefetch_layer_idx is not None:
+            assert prefetch_layer_idx in self.layer_to_buffer, f"Layer {prefetch_layer_idx} to be prefetched not found in registered prefteched layers"
```

Ruff (0.12.2)

Line too long (151 > 120)

(E501)